src/core/utils/prevent-screenshot.ts

```diff
@@ -0,0 +1,17 @@
+import { NativeModules } from 'react-native';
```

Comment:
also the file name should be `screenshot` since it also has an allow function :P